tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{39,310,311}
+envlist = py{310,311,312}
```

Comment:
This is crying for a separate commit. Also, please keep 3.9 supported.

Reply:
I'll revert this